sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -0,0 +1,547 @@
+      serializer: serializer,
+      shouldParseResponse: false
+    };
+    webResource.atomXmlOperationSpec = atomXmlOperationSpec;
+
+    return this.sendRequest(webResource);
+  }
+
+  /**
+   * Gets a resource.
+   *
+   * @param {string}             path                                                The resource path.
+   * @param {ResourceSerializer} serializer                                          The XML serializer to use.
+   * @return {Promise<HttpOperationResponse>}
+   */
+  private async _getResource(
+    path: string,
+    serializer: ResourceSerializer
+  ): Promise<HttpOperationResponse> {
+    let webResource: WebResource = new WebResource();
+
+    webResource.method = "GET";
+    webResource.url = this.getUrl(path);
+
+    const atomXmlOperationSpec: AtomXmlOperationSpec = {
+      serializer: serializer,
+      shouldParseResponse: true
+    };
+    webResource.atomXmlOperationSpec = atomXmlOperationSpec;
+
+    return this.sendRequest(webResource);
+  }
+
+  /**
+   * Lists resources.
+   *
+   * @param {string}             path                                                The resource path.
+   * @param {object}             requestBody                                         The resource handler.
+   * @param {ResourceSerializer} serializer                                          The XML serializer to use.
+   * @return {Promise<HttpOperationResponse>}
+   */
+  private async _listResources(
+    path: string,
+    requestBody: any,
+    serializer: ResourceSerializer
+  ): Promise<HttpOperationResponse> {
+    let webResource: WebResource = new WebResource();
+
+    webResource.method = "GET";
+
+    const queryParams: any = {};
+    if (requestBody) {
+      if (requestBody.skip) {
+        queryParams["$skip"] = requestBody.skip;
+      }
+      if (requestBody.top) {
+        queryParams["$top"] = requestBody.top;
+      }
+    }
+
+    webResource.url = this.getUrl(path, queryParams);
+
+    console.log(webResource.url);
+
+    const atomXmlOperationSpec: AtomXmlOperationSpec = {
+      serializer: serializer,
+      shouldParseResponse: true
+    };
+    webResource.atomXmlOperationSpec = atomXmlOperationSpec;
+
+    return this.sendRequest(webResource);
+  }
+
+  /**
+   * Lists resources.
+   *
+   * @param {string}             path                                                The resource path.
+   * @return {Promise<HttpOperationResponse>}
+   */
+  private async _deleteResource(path: any): Promise<HttpOperationResponse> {
+    let webResource: WebResource = new WebResource();
+
+    webResource.method = "DELETE";
+    webResource.url = this.getUrl(path);
+
+    return this.sendRequest(webResource);
+  }
+
+  private getUrl(path: string, queryParams?: any) {
+    const baseUri = `https://${this.endpoint}.servicebus.windows.net/${path}`;
+
+    const requestUrl: URLBuilder = URLBuilder.parse(baseUri);
+    requestUrl.setQueryParameter(`api-version`, `2017-04`);
```

Comment:
Shouldn't we be using the constants `API_VERSION_QUERY_KEY` and `CURRENT_API_VERSION` here?